### ISSUE TITLE
Move all RC applications to the same load balancer

### ISSUE THREAD

**Issue by adinuca** [NRGI/resourcecontracts.org#1116]
## Why
To decrease the costs for AWS we can configure one ALB to serve all RC applications. This would save approximately $80/months from the AWS bill.

## What
- [x] All targetgroups for RC are moved to resource-contracts-lb 

## Notes
This might cause a few minutes of downtime for each of the applications. The RC team should be notified before moving them.

**Comment by adinuca** [comment]
api.resourcecontracts.org(and api-production.resourcecontracts.org) and admin.resourcecontracts.org have been moved to the new load balancers.

index.resourcecontracts.org and resourcecontracts.org(and subsites) still need to be moved.

**Comment by adinuca** [comment]
Leaving resourcecontracts.org as it is because there are domains which are not handled in AWS and point to the rc-subsites application.

**Comment by charlesyoung** [comment]
@adinuca can I close this ticket?

**Comment by adinuca** [comment]
Hi @charlesyoung , I haven't done index.resourcecontracts.org. I need to check if I can do it without causing any downtime.

**Comment by adinuca** [comment]
@charlesyoung , I will move this for next week. I will close it once it is done.

**Comment by charlesyoung** [comment]
Thanks @adinuca

**Comment by adinuca** [comment]
We have decreased the number of load balancers used from 13 to 5 and the costs with approximately $150/month.

RC index application has not been moved as it doesn't work properly if 2 instances are started in the same time. Will raise a separate issue for it.
